backend/src/functions/opportunity/index.js

```
@@ -0,0 +1,78 @@
+            'Content-Type': 'application/json',
+            'Access-Control-Allow-Origin': '*',
+          },
+          body: JSON.stringify({
+            opportunities: [],
+            message: 'Get opportunities endpoint',
+          }),
+        };
+        
+      case 'POST':
+        // TODO: Implement creating opportunity in Salesforce
+        const body = JSON.parse(event.body || '{}');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)